gammapy/datasets/io.py

```diff
@@ -472,3 +476,180 @@ def read(self):
             )
 
         return SpectrumDatasetOnOff(name=name, exposure=exposure, **kwargs)
+
+
+class FermipyDatasetsReader(DatasetReader):
```

Comment:
Some Fermi-LAT user are not using Fermipy but only the Fermitools with which you define your own name for each file. So it is probably a good idea to support that. 
One thought would be to init the class with a filename for each file (i.e. counts_file, exposure_file, etc.) and create a classmethod `from_fermipy_config` that would init the class as you currently do.

Reply:
Now `create_dataset` takes each file names as input and its a static method so it can be used without init.
The init keep only the fermipy config file for simplicity.

Reply:
I see, clever !
What about the name though ? Do we want to keep Fermipy or simply Fermi ?

Reply:
I think the current implementation is good. Have some default to fermipy and allow other names.